commit/core.pyx

```diff
@@ -409,7 +409,7 @@ cdef class Evaluation :
         self.DICTIONARY['IC']['nF']    = self.DICTIONARY['TRK']['norm'].size
 
         # reorder the segments based on the "v" field
```

Comment:
Update the comment

Reply:
What do you mean with 'update the comment'? isn't clear enough?

Reply:
Maybe I do not understand, 
the comment was for `argsort` with `v`; now you are using `lexsort` with `o` and `v`. 
I that the case?

Reply:
Okay, I changed it mentioning that is with `v` and then with `o`.

Reply:
Is the order important? is that the order?